users-service/service.yml

```diff
@@ -26,4 +26,7 @@ messages:
 
 
 dependencies:
-  - 'mongo'
+  mongo:
+    prod: 'ds143588.mlab.com:43588'
+    dev: 'mongo:27017'
+    test: 'localhost:27017'
```

Comment:
Let's go with using dedicated MongoDB Docker images, and not running Mongo inside the  Docker images of other services. This means tests need to run via `exo-test`.